Sources/PackageDescription/Version.swift

```diff
@@ -25,13 +25,17 @@ public struct Version {
     }
 
     public init?(_ characters: String.CharacterView) {
-        let components = characters.split(".", maxSplit: 2, allowEmptySlices: true).map(String.init).flatMap{ Int($0) }.filter{ $0 >= 0 }
+        let components = Version.componentsFromCharacters(characters)
         guard components.count == 3 else { return nil }
 
         self.major = components[0]
         self.minor = components[1]
         self.patch = components[2]
     }
+    
+    private static func componentsFromCharacters(characters: String.CharacterView) -> [Int] {
+        return characters.split(".", maxSplit: 2, allowEmptySlices: true).flatMap{ Int(String($0)) }.filter{ $0 >= 0 }
```

Comment:
Ok I was mistaken. The old version on line 28 returned `[1]`, whereas the old version on line 168 returned `[1, 0, 0]` (because it didn't have the `allowEmptySlices` parameter. So I now see that all you were saying is that we need to keep the parameter and that it's not equivalent to omit it, as I suggested. Thanks for the correction!